Software/scan_key/AuthenticationSQL.py

```diff
@@ -24,19 +24,21 @@ def machineAuth(id, cursor):
     for auth in authdata:
         if(auth[machineType] != 1):
             print ("USER IS NOT AUTHORIZED\n")
+            os.system("omxplayer deny.wav &")
             errorSQL(id, 1)
             return (True)
 
     os.system("rm tmp")
```

Comment:
the tmp file is a way for me to extract the name of the machine. I create a tmp file, extract the name from the file, and remove the file. It creates an extra process, but it does the job.